src/inspector_agent.h

```diff
@@ -33,6 +35,9 @@ class Agent {
   // Stop the inspector agent
   void Stop();
 
+  void ContextCreated(const v8_inspector::V8ContextInfo& info);
```

Comment:
i saw the Threads bit in the new devtools, is that the purpose of the `contextGroupId`? Should we be including that as well?